File: Sample/JBSample/src/m/k/android/sample/jbsample/MainActivity.java
package m.k.android.sample.jbsample;

import m.k.android.sample.jbsample.activityoptions.ActivityOptionsActivity;
import m.k.android.sample.jbsample.notification.NotificationActivity;
import m.k.android.sample.jbsample.wallpaper.WallPaperActivity;
import m.k.android.sample.jbsample.webview.WebViewActivity;
import m.k.android.sample.jbsample.widget.WidgetManagerActivity;
import android.os.Bundle;
import android.app.Activity;
import android.content.Intent;
import android.view.Menu;
import android.view.View;

public class MainActivity extends Activity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.activity_main, menu);
        return true;
    }
    
    public void onClick(View view) {
    	switch(view.getId()) {
    		case R.id.btn1:
    			moveNotificationActivity();
    			break;
    		case R.id.btn2:
    			moveWidgetManagerActivity();
    			break;
    		case R.id.btn3:
    			moveWebViewActivity();
    			break;
    		case R.id.btn4:
    			moveActivityOptionsActivity();
    			break;
    		case R.id.btn5:
    			moveWallPaperActivity();
    	}
    }
    
    private void moveNotificationActivity() {
    	moveActivity(NotificationActivity.class);
    }
    
    private void moveWidgetManagerActivity() {
    	moveActivity(WidgetManagerActivity.class);
    }
    
    private void moveWebViewActivity() {
    	moveActivity(WebViewActivity.class);
    }

    private void moveActivityOptionsActivity() {
    	moveActivity(ActivityOptionsActivity.class);
    }
    
    private void moveWallPaperActivity() {
    	moveActivity(WallPaperActivity.class);
    }
    
    private void moveActivity(Class<?> actClass) {
    	Intent intent = new Intent(this, actClass);
    	startActivity(intent);
    }
}

File: Sample/MultipleDeviceSupportSample/app/src/main/java/com/github/mag0716/multipledevicesupportsample/ui/MainActivity.kt
package com.github.mag0716.multipledevicesupportsample.ui

import android.content.Intent
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import com.github.mag0716.multipledevicesupportsample.R
import com.github.mag0716.multipledevicesupportsample.ui.base.LoggingActivity
import com.github.mag0716.multipledevicesupportsample.ui.license.LicenseActivity
import com.github.mag0716.multipledevicesupportsample.ui.settings.SettingsActivity

class MainActivity : LoggingActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
        menuInflater.inflate(R.menu.menu, menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            R.id.settings -> {
                startActivity(Intent(this, SettingsActivity::class.java))
            }
            R.id.license -> {
                startActivity(Intent(this, LicenseActivity::class.java))
            }
        }
        return false
    }
}